archive/multisource.go

```diff
@@ -19,65 +18,58 @@ import (
 	"github.com/brimsec/zq/zng/resolver"
 	"github.com/brimsec/zq/zqd/api"
 	"github.com/brimsec/zq/zqe"
+	"go.uber.org/multierr"
 )
 
-type multiCloser struct {
-	closers []io.Closer
+type Source interface {
```

Comment:
Calling this Source feels odd given that it embeds a driver.MultiSource and is returned by NewMultiSource.
```suggestion
type MultiSource interface {
```

Reply:
sg